### DISCUSSION TITLE
Cloudinary: Uploading assets and keeping their original filenames

### DISCUSSION THREAD

**Discussion by luuhung93** [keystonejs/keystone#7758] [Feature requests]
Uploading assets and keeping their original filenames.
https://support.cloudinary.com/hc/en-us/articles/202520762-Uploading-assets-and-keeping-their-original-filenames
This good for Seo.

**Comment by dcousens** [comment]
@luuhung93 is this a question or a feature request?

**Reply by luuhung93** [reply]
yep, a feature request.

**Comment by luuhung93** [comment]
when can I have this feature option on keystone?

**Reply by flexdinesh** [reply]
We don't have this in our roadmap at the moment. We'll be happy to review a PR if you're keen to contribute.
